app/components/signin_modal/view.jsx

```diff
@@ -30,7 +30,7 @@ export default class SigninModal extends React.Component {
     let valid = true;
 
     Object.keys(this.props.user).forEach((key) => {
-      if (this.props.user[key].trim().length < 6) valid = false;
+      if (this.props.user[key].length < 6) valid = false;
```

Comment:
just wondering, what is a purpose of the trick? Why do we need to check all keys length to be at least 6 characters?